methods.py

```diff
@@ -1353,7 +1353,7 @@ def format_key_value(v):
                 vsconf = f'{target}|{a["platform"]}'
                 break
 
-        condition = "'$(GodotConfiguration)|$(GodotPlatform)'=='" + vsconf + "'"
```

Comment:
This is a variable name, not the variable itself. Please do not change these so we can avoid merge conflicts. This goes for two other files as well.

Reply:
will revert all the variable names in the squashed commit